### PR TITLE
Split utils

### LINKED ISSUE #86: Split deployer/utils into deployer/internal and deployer/utils - only utils are intended for external use and require simple primitives as input

### PR DESCRIPTION
**Pull request by ajsantander**
Fix #86

### CODE REVIEW

File: packages/core-js/utils/contracts.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+  return sourceBytecodeHash === remoteBytecodeHash;
+}
+
+function getBytecodeHash(bytecode) {
+  return ethers.utils.sha256(bytecode);
+}
+
+async function getRemoteBytecode(address) {
+  return await ethers.provider.getCode(address);
```
Comment (ajsantander) [inline packages/core-js/utils/contracts.js:17]: Would be better not to use ethers default provider

File: packages/deployer/subtasks/prepare-deployment.js [hunk 2]
```diff
@@ -52,7 +52,7 @@ subtask(
  * @returns {{ currentFile: string, previousFile: string }}
  */
 async function _determineDeploymentFiles(deploymentsFolder, alias) {
-  const deployments = getDeploymentFiles(deploymentsFolder);
+  const deployments = getAllDeploymentsFilesForInstance(deploymentsFolder);
   const previousFile = deployments.length > 0 ? deployments[deployments.length - 1] : null;
```
Comment (ajsantander) [inline packages/deployer/subtasks/prepare-deployment.js:56]: Rename to latest?

File: packages/core-js/utils/contracts.js (hunk 1 above)
Comment (ajsantander) [reply packages/core-js/utils/contracts.js:17]: Fixed

File: packages/deployer/subtasks/prepare-deployment.js (hunk 2 above)
Comment (ajsantander) [reply packages/deployer/subtasks/prepare-deployment.js]: Fixed

**Review by leomassazza — APPROVED** [review]
👍

**Pull request merged**